state/machine.go

```diff
@@ -2104,6 +2092,61 @@ func (m *Machine) CreateUpgradeSeriesPrepareLock() error {
 	return nil
 }
 
+// RemoveUpgradeSeriesPrepareLock remove a series upgrade prepare lock for a
+// given machine.
+func (m *Machine) RemoveUpgradeSeriesPrepareLock() error {
+	buildTxn := func(attempt int) ([]txn.Op, error) {
+		if attempt > 0 {
+			if err := m.Refresh(); err != nil {
+				return nil, errors.Trace(err)
+			}
+		}
+		locked, err := m.IsLocked()
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		if !locked {
+			return nil, errors.NotFoundf("upgrade series prepare lock for machine %q", m)
```

Comment:
It probably shouldn't be called a "prepare lock" it is the UpgradeSeries lock isn't it?
It sounds funny to say the "prepare lock wasn't found" when removing it.
You are locking the machine for upgrade, and 'prepare' is just the step that happens when you do so.

A similar issue is in the message for Create.*Lock. when the lock already exists.